modules/github.py

```diff
@@ -60,6 +133,35 @@ async def on_message(self, message: discord.Message) -> None:
 
             await message.channel.send(GITHUB_ISSUE_URL.format(lib, issue))
 
+        codeSegment = await self.format_highlight_block(message.content)
+
+        if codeSegment is None:
+            return
+
+        await message.add_reaction(self.code_highlight_emoji)
+
+        path = codeSegment['path']
+        _min = codeSegment['min']
+        _max = codeSegment['max']
+        code_fmt = codeSegment['msg']
+
+        def check(reaction, user):
+            return reaction.emoji == self.code_highlight_emoji and user != self.bot.user \
+                and message.id == reaction.message.id
+
+        await self.bot.wait_for("reaction_add", check=check)
```

Comment:
`Bot.wait_for` should be wrapped in a `try: except:` block which handles `asyncio.TimeoutError`, with the exception being raised when the timeout is hit.

There should also be a timeout added, that wait we don't have N pending waiters.